test/pretty-print.test.js

```diff
@@ -364,3 +364,31 @@ test('pretty print - includeMeta, includeHooks', (t, done) => {
     done()
   })
 })
+// test/printRoutes-commonPrefix.test.js
+
+const assert = require('node:assert')
```

Comment:
Let's use the `assert` that comes with the `TestContext` (the variable passed to the `test` callback.